### PR TITLE
Mount JSON config as file instead of in environment

### PR DESCRIPTION
**Pull request by GordonDiggs**
Dependent on https://github.com/codeclimate/spec/pull/1

/cc @codeclimate/review 

- bundler-audit: https://github.com/codeclimate/codeclimate-bundler-audit/pull/4
- coffeelint: https://github.com/codeclimate/codeclimate-coffeelint/pull/2
- csslint: https://github.com/codeclimate/codeclimate-csslint/pull/5
- eslint: https://github.com/codeclimate/codeclimate-eslint/pull/14
- gofmt: https://github.com/codeclimate-community/codeclimate-gofmt/pull/2
- golint: https://github.com/codeclimate-community/codeclimate-golint/pull/2
- govet: https://github.com/codeclimate-community/codeclimate-govet/pull/2
- phpcodesniffer: https://github.com/codeclimate/codeclimate-phpcodesniffer/pull/7
- rubocop: https://github.com/codeclimate/codeclimate-rubocop/pull/4
- rubymotion: https://github.com/HipByte/codeclimate-rubymotion/pull/2
- watson: https://github.com/codeclimate-community/codeclimate-watson/pull/2

### CODE REVIEW

**Comment by pbrisbin** [comment]
LGTM - is this a hanging yellow?

**Comment by GordonDiggs** [comment]
@codeclimate/review this is ready for another look! I added a line to the Makefile to update all engines that are installed on the user's machine

**Comment by ABaldwinHunter** [comment]
LGTM -- but maybe wait for other feedback. :)

File: Makefile [hunk 1]
```diff
@@ -3,6 +3,7 @@ PREFIX ?= /usr/local
 install:
 	bin/check
 	docker pull codeclimate/codeclimate:latest
+	docker images | grep codeclimate/codeclimate- | awk '{print $1}' | while read image; do docker pull $image; done
```
Comment (pbrisbin) [inline Makefile]: How about

```
... | xargs docker pull
```

Comment (GordonDiggs) [inline Makefile]: Yep that would work much better!

Comment (pbrisbin) [inline Makefile]: Generally `grep <re> | awk { <cmds> }` can be replaced with `awk /<re>/ { <cmds> }`

```
... | ' awk /codeclimate\/codeclimate-/ { print $1 }' ...
```

Comment (GordonDiggs) [inline Makefile]: @pbrisbin updated!

**Comment by pbrisbin** [comment]
LGTM

**Pull request merged**